test/Generator/ParameterGeneratorTest.php

```diff
@@ -584,6 +584,18 @@ public function testGetInternalClassDefaultParameterValue()
         self::assertSame('null', strtolower((string) $parameter->getDefaultValue()));
     }
 
+    /**
+     * @group laminas/laminas-code#53
+     */
+    public function testGetInternalClassDefaultParameterValueWithUnionType()
+    {
+        $parameter = ParameterGenerator::fromReflection(new ParameterReflection([\Phar::class, 'extractTo'], 1));
```

Comment:
This builds an implicit dependency to `ext-phar` that I'd like to avoid.

Instead, we should have our own (stable) test asset.

I'll do this locally though :+1:

Reply:
Adding this tests with a local class in a phpt file to avoid syntax on PHP <= 7.4 error is ok for you ?

Reply:
That's what I'm currently doing - creating new test assets with PHP8.0-only features.

I'm cherry-picking your branch in my work :+1:

Reply:
Included your diff in #57